extensions/amp-sidebar/0.1/amp-sidebar.css

```diff
@@ -61,3 +61,8 @@ amp-sidebar[side] {
   background-color: #000;
   z-index: 2147483646;
 }
+
+.amp-sidebar-mask {
+  opacity: 0.2;
```

Comment:
I am okay with changing opacity, i think we should lock the margin, padding , outline, display, float,  z-index and visibility using !important on the private selector above. 
The reason i have not approved this is because i am still nervous about creating a public class. for the mask. 
I think this is making it too complicated. 

CC @jridgewell